openhcl/underhill_core/src/nvme_manager.rs

```diff
@@ -217,6 +228,8 @@ struct NvmeManagerWorker {
     driver_source: VmTaskDriverSource,
     #[inspect(iter_by_key)]
     devices: HashMap<String, nvme_driver::NvmeDriver<VfioDevice>>,
+    #[inspect(iter_by_key)]
+    controller_instance_ids: HashMap<String, Guid>,
```

Comment:
I'm not a big fan of having a separate HashMap here, but I didn't get a good feeling about other approaches I thought of: using a two-part key to the devices HashMap (controller ID + PCI ID) or pushing the controller ID through to the NVMe device struct in a different file. I am open to suggestions of better approaches 😃

Reply:
Thanks Eric! I would agree with you.

We should make this a field of the `NvmeDriver`. You can then use that in the nvme manager, and we can also log the guid in the driver code itself. Perhaps call it `name` or `host_id` (something generic that keeps meaning if we support things other than `vpci`-backed devices.)

Perhaps this is a bit unfortunate, since we're perpetuating the vpci concept to something that really doesn't need it (a `VfioDevce`), but I think we will find ourselves wanting the diagnostic info.